### PR TITLE
CSP Push One

### PR DESCRIPTION
**Pull request by cocoacacao**
- Replaces Montserrat with Poiret One

- Adds Anglo-Saxon Runes font to be used on hover for clickable menu elements (broken)

- Makes the entirety of the email buttons on the Contact subpage clickable (works but the link style applies to the whole button, kinda ugly)

- Adds the social media subheader to every subpage

- Reworks the subpage buttons on the index page to be centered around the logo; fine, except the middle line is centered on the entirety of the middle rather than the logo, so it looks awful

- Adds a slot for Vocazone on the about subpage, still requires a blurb

- Replaces all PNG/JPG files with higher quality and MUCH smaller .webp files

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -79,15 +85,32 @@ <h4 class="header">Social Media</h4>
             <p>
                 <!-- Twitter -->
                 <a href="https://twitter.com/AlderS_Official">
-                    <img class="media-icon" src="asset/twitter_logo.png" alt="Twitter">
+                    <img class="media-icon" src="asset/twitter_logo.webp" alt="Twitter">
```
Comment (vocatart) [inline about.html:88]: dupe?

Comment (cocoacacao) [reply about.html:88]: .png -> .webp

Comment (vocatart) [reply about.html:88]: i am aware.

File: contact.html [hunk 2]
```diff
@@ -35,35 +35,44 @@ <h4 class="header">Contacting Us</h4>
...
+			<div class="email-separator">
+			</div>
+			<a href="mailto:prom@alder-s.com">
             <div class="email-box">
                 <p>For marketing, contact:</p>
-                <a href="mailto:prom@alder-s.com">promo@alder-s.com</a>
+                promo@alder-s.com
             </div>
+			</a>
         </div>
```
Comment (vocatart) [inline contact.html:60]: revert this change for now

File: fonts.css [hunk 3]
```diff
@@ -1,27 +1,13 @@
 @font-face {
```
Comment (vocatart) [inline fonts.css:1]: PLEASE RETURN MY FONT HANDLING??????

File: index.html [hunk 4]
```diff
@@ -3,28 +3,43 @@
...
+				<div class="menu-button">
+					<a href="merch">Merchandise</a>
+				</div>
+		</div>
+				
+		<div class="menu-container">				
+				<div class="menu-button">
+					<a href="contact">Contact</a>
+                </div>
+ 		</div>
```
Comment (vocatart) [inline index.html:43]: revert this change

File: merch.html [hunk 5]
```diff
@@ -3,9 +3,9 @@
     <head>
         <title>Alder S - Merchandise</title>
         <link rel="stylesheet" type="text/css" href="style.css">
-        <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
-        <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
-        <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+        <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.webp">
+        <link rel="icon" type="image/webp" sizes="32x32" href="/favicon-32x32.webp">
+        <link rel="icon" type="image/webp" sizes="16x16" href="/favicon-16x16.webp">
         <link rel="manifest" href="/site.webmanifest">
```
Comment (vocatart) [inline merch.html:9]: fix this (again)

File: style.css [hunk 6]
```diff
@@ -4,33 +4,43 @@
```
Comment (vocatart) [inline style.css:4]: revert any changes that were mentioned in the HTML file

**Review by vocatart — CHANGES_REQUESTED** [review]
See comments

**Comment by vocatart** [comment]
@cocoacacao please make sure that you're also formatting and commenting the code correctly xoxo

File: fonts.css (hunk 3 above)
Comment (cocoacacao) [reply fonts.css:1]: no but i'll add mulish it's better anyway

File: contact.html (hunk 2 above)
Comment (cocoacacao) [reply contact.html:60]: ok but i'm leaving the separators

Comment (vocatart) [reply contact.html:60]: please revert this completely as I'm gonna look over this one, id rather have you submit this as an issue for issue handling.

Comment (cocoacacao) [reply contact.html:60]: i don't know what you're talking about so I'm not going to

File: index.html (hunk 4 above)
Comment (cocoacacao) [reply index.html:43]: doing the horizontal thing instead

File: style.css (hunk 6 above)
Comment (cocoacacao) [reply style.css:4]: OKAY BUT LIKE PLEASE FIX THE HOVER THINGS I DON'T KNOW HOW THEY WORK

Comment (vocatart) [reply style.css:4]: REVERT THE CHANGES FIRST

File: contact.html (hunk 2 above)
Comment (vocatart) [reply contact.html:60]: ??????

**Comment by vocatart** [comment]
Closing this pr in the hopes that the next one will be neater for us to collab on (this is both of our faults I should've helped), submit any changes you want to work on to the issues tab so we can triage them together and then make a more comprehensive pr! i feel bad for denying this but ill leave the branch here for you to reference code from haha

**Review by vocatart — COMMENTED** [review]
closing pr
